### PR TITLE
Create homepage layout

### PR DESCRIPTION
**Pull request by ac1214**

### CODE REVIEW

File: src/components/App/NavBar.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { useSelector } from 'react-redux';
+import { Navbar, DropdownButton, Dropdown, Form, FormControl, Button } from 'react-bootstrap';
+
+const NavBar = () => {
+  const loggedIn = useSelector((state) => {
+    return state.userReducer.loggedIn;
+  });
+
+  function ProfileButton(props) {
+    const isLoggedIn = props.isLoggedIn;
```
Comment (satyakig) [inline src/components/App/NavBar.jsx]: Remove this line

File: src/components/App/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    if (isLoggedIn) {
```
Comment (satyakig) [inline src/components/App/NavBar.jsx]: just read the above `loggedIn` variable

File: src/components/App/NavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Navbar, DropdownButton, Dropdown, Form, FormControl, Button } from 'react-bootstrap';
+
+const NavBar = () => {
+  const loggedIn = useSelector((state) => {
+    return state.userReducer.loggedIn;
+  });
+
+  function ProfileButton(props) {
```
Comment (satyakig) [inline src/components/App/NavBar.jsx]: rename to `profileButton` to name it like a function and remove the props argument

File: src/components/App/NavBar.jsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        <Form inline className="mt-2 mt-sm-0">
+          <FormControl type="text" placeholder="Search" />
+        </Form>
+      </Navbar.Collapse>
+      <Navbar.Collapse className="justify-content-end">
+        <Button variant="outline-primary" type="create-event" className=" mr-2 mt-2 mt-sm-0">
+          Create Event
+        </Button>
+
+        <ProfileButton isLoggedIn={loggedIn} />
```
Comment (satyakig) [inline src/components/App/NavBar.jsx]: change this to `{profileButton()}`

File: src/components/App/App.jsx [hunk 5]
```diff
@@ -11,20 +13,8 @@ const App = () => {
 
   return (
```
Comment (satyakig) [inline src/components/App/App.jsx]: This component needs to set the heights and widths for Navbar and HomePage. and make sure they size in mobile and nothing overflows once HomePage starts growing

File: src/redux/reducers/userReducer.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import { UPDATE_USER } from '../actions/UserActions';
```
Comment (Sadat21) [inline src/redux/reducers/userReducer.js]: File should be called UserReducer not userReducer

File: src/redux/actions/UserActions.js [hunk 7]
```diff
@@ -0,0 +1,8 @@
+export const UPDATE_USER = 'UPDATE_USER';
```
Comment (Sadat21) [inline src/redux/actions/UserActions.js]: I don't think any of these redux files are needed for this PR. Most of it is going to be overwritten by the authentication PR

**Comment by Yang-Andy** [comment]
can you add a screenshot of what it looks like.

**Comment by ac1214** [comment]
> can you add a screenshot of what it looks like.

Screenshot from 2020-02-27 20-06-16

**Pull request merged**